CHANGELOG.rst

```diff
@@ -2,7 +2,11 @@ Unreleased
 ==========
 
 Changes since 0.4:
+
+* Added the opt-in classic_division fixer.
 * Updated the ``dict_six`` fixer to support ``six.viewitems()`` and friends.
+* Documentation corrections.
```

Comment:
Does this point refer to the change between #109 and this PR? I don't think there's any need to list it if the only corrections are to documentation added since 0.4.